source/Userdocs/Software/jNeuroML.md

```diff
@@ -1,3 +1,55 @@
+(jneuroml)=
 # jNeuroML
 
-jNeuroML is a comprehensive Java application for handling LEMS and NeuroML 2. It bundles jLEMS together with the LEMS definitions for NeuroML 2 ComponentTypes, and can simulate any LEMS model as well as many NeuroML 2 models, validate NeuroML 1/2, and export to and import from a number of other simulator formats.
+jNeuroML is a comprehensive application built in Java for handling LEMS and NeuroML 2.
+
+It bundles {ref}`jLEMS <jLEMS>` together with the LEMS definitions for {ref}`NeuroML 2 ComponentTypes <userdocs:neuromlv2>`. The aim is to
+allow easy access to a range of NeuroML2/LEMS functionality through a simple command line interface (**jnml**), or in other Java based
+applications when used as a library.
+
+jNeuroML can:
+
+- **Validate** NeuroML v1.8.1 and v2.x files
+- **Simulate** most NeuroML 2 models (simulation should be specified in the [Simulation element in LEMS file](https://github.com/NeuroML/NeuroML2/blob/master/LEMSexamples/LEMS_NML2_Ex5_DetCell.xml))
+- **Export** NeuroML 2 and LEMS to many formats such as Neuron, Brian, Matlab, etc.
+- **Import** other languages into LEMS (e.g. SBML)
+
+## Quick start
+
+To get a precompiled binary for jNeuroML, go to [https://github.com/NeuroML/jNeuroML/releases](https://github.com/NeuroML/jNeuroML/releases), download the latest **jNeuroML.zip** file and you have everything you need. Unzip on your local machine and go into the *jNeuroMLJar* directory.
+
+You can also check out the most recent verson of the jar file using:
```

Comment:
I'd skip this here: this shouldn't be exposed to normal end users. We can put this in the dev docs section?